core/exception/ExceptionHandler.ts

```diff
@@ -300,7 +300,7 @@ export class ExceptionHandler {
     }
 
     private logException(e: HoistException, opts: ExceptionHandlerOptions) {
-        return opts.showAsError ? console.error(opts.message, e) : console.debug(opts.message);
+        return opts.showAsError ? logError([opts.message, e], this) : logDebug(opts.message, this);
```

Comment:
Let's look at a log statement from here (a handled exception) and see if this output is now better